cra-battery-considerate-loading/README.md

```diff
@@ -0,0 +1,19 @@
+
+# Demo: Battery-considerate resource loading in React
+
+A demo showing how to use battery-considerate resource loading in React.
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Installation
+```
+git clone https://github.com/anton-karlovskiy/battery-considerate-loading.git
```

Comment:
Could we revise the paths to point to this repo?

Reply:
Whoops, yes my bad. Let me fix it right away.

Reply:
I'm also going to remove glitch part since URLs do not look professional.